public/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "short_name": "MOSIP Idp",
+  "name": "Create MOSIP Idp",
```

Comment:
Same as above

Reply:
name updated to MOSIP - OIDC Demo Health Portal